dts/arm/nordic/nrf52840.dtsi

```diff
@@ -137,6 +137,22 @@
 			interrupt-names = "wdt";
 			label = "WDT";
 		};
+
+		cryptocell: crypto@5002a000 {
+			compatible = "nordic,nrf-cc310";
+			reg = <0x5002A000 0x1000>;
```

Comment:
For style-consistency, please, use capital letters for hex values, here and inline 148

Reply:
I can't. If I change it I have following warnings:
```
nrf52840_pca10056.dts_compiled: Warning (simple_bus_reg): Node /soc/spi@4002B000 simple-bus unit address format error, expected "4002b000"
nrf52840_pca10056.dts_compiled: Warning (simple_bus_reg): Node /soc/crypto@5002A000 simple-bus unit address format error, expected "5002a000"
```